bin/cli.js

```diff
@@ -243,6 +245,7 @@ const getConfig = () => {
 
 const runNode = (config, cb) => {
   if (config.logLevel === 'quiet') metatests.runner.instance.removeReporter();
+  if (config.runTodo) metatests.runner.instance.runTodo();
```

Comment:
Shouldn't we also enable them for browser?

Reply:
It's ok, but I'd prefer to make all changes in a [dedicated package](https://github.com/metarhia/metatests-browser-runner/pull/1), so we don't have to repeat it twice.

Reply:
Could you add this option there then?
Also, that package is not yet released so we should support browser runner in this package as well.

Reply:
It seems that if todo tests fail it will also fail parent
```bash
$ ./bin/cli.js --browser --browsers=ChromeHeadless --log-level=error --run-todo test/imperative.js
(node:29468) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
⚠ ｢wdm｣: 
ℹ ｢wdm｣: Compiled with warnings.
ℹ ｢wdm｣: Compiling...
⚠ ｢wdm｣: 
ℹ ｢wdm｣: Compiled with warnings.
ℹ ｢wdm｣: Compiling...
⚠ ｢wdm｣: 
ℹ ｢wdm｣: Compiled with warnings.

HeadlessChrome 0.0.0 (Linux 0.0.0):
============================================================
------------------------------------------------------------
TODO Test succeeded: todo must not fail

      Type: strictSame
  Expected: 42 is number
    Actual: 13 is number
     Stack: Error
                ImperativeTest._check (http:/localhost:9876/base/build/adapter.js?a8b97a8ddd4a0e17541bc643e1eede8914a29887:38534:14)
                ImperativeTest.strictSame (http:/localhost:9876/base/build/adapter.js?a8b97a8ddd4a0e17541bc643e1eede8914a29887:38555:10)
                ImperativeTest.metatests.testSync.test [as func] (http:/localhost:9876/base/build/adapter.js?a8b97a8ddd4a0e17541bc643e1eede8914a29887:38927:8)
                funcDomain.run (http:/localhost:9876/base/build/adapter.js?a8b97a8ddd4a0e17541bc643e1eede8914a29887:38772:31)
                EventEmitter.d.run (http:/localhost:9876/base/build/adapter.js?a8b97a8ddd4a0e17541bc643e1eede8914a29887:14464:5)
                ImperativeTest.runNow (http:/localhost:9876/base/build/adapter.js?a8b97a8ddd4a0e17541bc643e1eede8914a29887:38772:16)
                process.nextTick (http:/localhost:9876/base/build/adapter.js?a8b97a8ddd4a0e17541bc643e1eede8914a29887:38755:12)
                Item.run (http:/localhost:9876/base/build/adapter.js?a8b97a8ddd4a0e17541bc643e1eede8914a29887:5737:14)
                drainQueue (http:/localhost:9876/base/build/adapter.js?a8b97a8ddd4a0e17541bc643e1eede8914a29887:5707:42)

HeadlessChrome 0.0.0 (Linux 0.0.0) ERROR
  {
    "message": "Uncaught Error\nat build/adapter.js:38673:5\n\nError\n    at ImperativeTest.bailout (build/adapter.js:38673:11)\n    at subtestCallbackEnd (build/adapter.js:38406:18)\n    at ImperativeTest._subtestCallback.test (build/adapter.js:38439:9)\n    at ImperativeTest.EventEmitter.emit (build/adapter.js:7422:20)\n    at ImperativeTest._end (build/adapter.js:22969:12)\n    at ImperativeTest._end (build/adapter.js:38712:11)\n    at ImperativeTest.end (build/adapter.js:22963:10)\n    at ImperativeTest.end (build/adapter.js:38691:11)\n    at process.nextTick (build/adapter.js:38759:60)\n    at Item.run (build/adapter.js:5737:14)",
    "str": "Uncaught Error\nat build/adapter.js:38673:5\n\nError\n    at ImperativeTest.bailout (build/adapter.js:38673:11)\n    at subtestCallbackEnd (build/adapter.js:38406:18)\n    at ImperativeTest._subtestCallback.test (build/adapter.js:38439:9)\n    at ImperativeTest.EventEmitter.emit (build/adapter.js:7422:20)\n    at ImperativeTest._end (build/adapter.js:22969:12)\n    at ImperativeTest._end (build/adapter.js:38712:11)\n    at ImperativeTest.end (build/adapter.js:22963:10)\n    at ImperativeTest.end (build/adapter.js:38691:11)\n    at process.nextTick (build/adapter.js:38759:60)\n    at Item.run (build/adapter.js:5737:14)"
  }
Metatests finished with code 1
```